src/uct/api/uct.h

```diff
@@ -2678,6 +2686,8 @@ ucs_status_t uct_rkey_unpack(uct_component_h component, const void *rkey_buffer,
  * This routine returns a local pointer to the remote memory
  * described by the rkey bundle. The MD must support
  * @ref UCT_MD_FLAG_RKEY_PTR flag.
+ * @note @ref UCT_MD_FLAG_RKEY_PTR is deprecated and replaced by the
+ * @ref UCT_COMPONENT_FLAG_RKEY_PTR component flag.
```

Comment:
let's replace lines 2687-2688 instead of adding a note

Reply:
maybe better keep it  showing that backward compatibility is not broken?

Reply:
i think the documentation should aim for new application and show the recommended/main way to use the API.
older application, that need the backward compat, are already written and don't need the API doc anymore.
if really important, we can add note that the component flag replaced the older MD flag.

Reply:
ok, fixed